sdk/cosmosdb/cosmos/samples-dev/typescript/Shared/handleError.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates using SasTokens for granting scoped access to Cosmos resources. *Private feature*
```

Comment:
This file should probably have `@azsdk-util true` and not `@summary`. That designator lets us treat it as a util file, and it won't be included in the sample index (the markdown table) by default.